src/eslint.ts

```diff
@@ -305,6 +322,12 @@ export class Eslint extends Component {
           '@typescript-eslint/parser': ['.ts', '.tsx'],
         },
         'import/resolver': {
+          ...( options.aliasMap && options.aliasExtensions && {
```

Comment:
Looking at https://www.npmjs.com/package/eslint-import-resolver-alias, I think aliasExtensions can be omitted, so it should probably be:
```suggestion
          ...( options.aliasMap && {
```